api/src/test/scala/org/globalnames/resolver/api/QueryParserSpec.scala

```diff
@@ -0,0 +1,118 @@
+package org.globalnames.resolver.api
+
+import org.scalatest.FunSpec
+import org.globalnames.resolver.api.QueryParser.{Modifier, SearchPart}
+
+class QueryParserSpec extends FunSpec {
+  val context = describe _
+
+  describe(".result") {
+    context("modifiers") {
+      it("works without modifier") {
+        assert(QueryParser.result("Pomatomus") ==
```

Comment:
So, what assert syntax we would be using? `shouldBe` is used in `GnresolverMicroserviceIntegrationSpec`, that I prefer.